pandas/formats/style.py

```diff
@@ -218,8 +218,8 @@ def format_attr(pair):
 
         cell_context = dict()
 
-        n_rlvls = self.data.index.nlevels
-        n_clvls = self.data.columns.nlevels
+        n_rlvls = 0 if self.data.empty else self.data.index.nlevels
```

Comment:
I think you'll have to change this around a bit. `DataFrame.empty` is True if either the index or columns is empty. So you get

```python
pd.DataFrame([], index=[1, 2]).empty  #  True
pd.DataFrame([], columns=['A', 'B']).empty  # True
```